nose.cfg

```diff
@@ -2,3 +2,13 @@
 detailed-errors=1
 where=yt
 exclude=answer_testing
+#with-answer-testing=1
+with-coverage=1
+#cover-branches=1
```

Comment:
Can you delete these two commented out lines?

Reply:
I wanted to keep these configurations, as later on I want to check if each build could be run with answer_testing.

Reply:
In general we don't like leaving behind commented out code as it can be confusing to future readers. I'd create a separate reminder for yourself and then when you come back to integrating with answer testing you could add these lines back in in a subsequent pull request.

Reply:
That sounds good to me! Will do.